<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workorder_list_dlg</class>
 <widget class="QDialog" name="workorder_list_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>ERP系统</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>云端系统</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="workorder_lb">
        <property name="text">
         <string>工单号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="workorder_edt"/>
      </item>
      <item>
       <widget class="QLabel" name="mi_nums_lb">
        <property name="text">
         <string>微逆总数：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>148</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="read_pb">
        <property name="text">
         <string>从系统读取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="write_pb">
        <property name="text">
         <string>录入云端系统</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mi_list_lb">
        <property name="text">
         <string>微逆连续序列号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="start_mi_edt"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>~</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stop_mi_edt"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>连续编号外的额外微逆编号</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="a_table_wd"/>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>2</number>
            </property>
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="a_mi_lb">
              <property name="text">
               <string>微逆编号</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="a_mi_edt"/>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="a_plus_pb">
              <property name="text">
               <string>增加</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="a_minus_edt">
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>连续编号内缺失微逆编号</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="l_table_wd"/>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QFormLayout" name="formLayout_2">
            <property name="horizontalSpacing">
             <number>2</number>
            </property>
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="l_mi_lb">
              <property name="text">
               <string>微逆编号</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="l_mi_edt"/>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="l_plus_pb">
              <property name="text">
               <string>增加</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="l_minus_edt">
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>464</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ok_pb">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cache_pb">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
